docs/tryitnow.rst

```diff
@@ -1,9 +1,17 @@
 Try It Now!
 ===========
-Try out xonsh right here in the browser. Just press the triangle play button in
-the bottom right of the `Repl.it <https://repl.it>`_ frame below.
-**Note:** this is not a fully featured xonsh instance due to repl.it limitations.
+Try out xonsh right here in the browser. Just press the "Run This"
+button below!
 
 .. raw:: html
 
-    <iframe height="400px" width="100%" src="https://repl.it/@scopatz/xonsh?lite=true" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
+    <div id="tryitnow"></div>
+    <script>
+      var app = Elm.Main.init({
+        node: document.getElementById('tryitnow'),
+          flags: {
+            placeholder: "<img src=\"_static/better_colors_windows.png\">",
+            serverUrl: "https://runthis.xonsh.org:80",
```

Comment:
This is extremely funky. Why are you running HTTPS on port 80?

Reply:
Yeah, I don't know why it doesn't resolve otherwise.  I would love help debugging it.